{"uuid":"a579d3d4-f681-4024-a17a-10bf4103067e","historyId":"cb2eb943d3996a09fffd36bca90828a0","testCaseId":"1a73765ccf7bf07909a1a976e1b39492","fullName":"src/test/resources/features/linkedList.feature:25","labels":[{"name":"tag","value":"wip"},{"name":"host","value":"Blossoms"},{"name":"thread","value":"49816@Blossoms.main(1)"},{"name":"feature","value":"Linked List functionality"},{"name":"story","value":"validating Run function with valid code and invalid code"},{"name":"suite","value":"Linked List functionality"},{"name":"testClass","value":"validating Run function with valid code and invalid code"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/bloss/src/DsAlgoProject/src/test/resources/features/linkedList.feature"},{"name":"package","value":"src.test.resources.features.linkedList_feature.Linked List functionality"}],"links":[],"name":"validating Run function with valid code and invalid code","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Unexpected outcome specified in test case: SyntaxError: bad input on line 1","trace":"java.lang.AssertionError: Unexpected outcome specified in test case: SyntaxError: bad input on line 1\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat stepDefinitions.LinkedListSteps.the_user_gets_the_message(LinkedListSteps.java:82)\r\n\tat ✽.the user gets the message 'SyntaxError: bad input on line 1'(file:///C:/Users/bloss/src/DsAlgoProject/src/test/resources/features/linkedList.feature:22)\r\n"},"stage":"finished","steps":[{"name":"Given the user is logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733161610379,"stop":1733161612684},{"name":"Given the user is in the python editor of Introduction page","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733161612684,"stop":1733161613288},{"name":"When the user enters 'print(\\\"Hello\\\")abc' in the python editor and clicks Run","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733161613289,"stop":1733161613290},{"name":"Then the user gets the message 'SyntaxError: bad input on line 1'","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Unexpected outcome specified in test case: SyntaxError: bad input on line 1","trace":"java.lang.AssertionError: Unexpected outcome specified in test case: SyntaxError: bad input on line 1\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat stepDefinitions.LinkedListSteps.the_user_gets_the_message(LinkedListSteps.java:82)\r\n\tat ✽.the user gets the message 'SyntaxError: bad input on line 1'(file:///C:/Users/bloss/src/DsAlgoProject/src/test/resources/features/linkedList.feature:22)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1733161613290,"stop":1733161613293}],"attachments":[],"parameters":[{"name":"code","value":"print(\\\"Hello\\\")abc"},{"name":"expectedOutcome","value":"SyntaxError: bad input on line 1"}],"start":1733161610376,"stop":1733161613458}